tests/test_post_proc_plugin.py

```diff
@@ -70,6 +70,7 @@ def test_missing_subclass():
     with pytest.raises(McsUserError, match=r'No subclass of PostProcessor .*'):
         PostProcessor.load_plugin(path)
 
+@pytest.mark.skip()
```

Comment:
@mbarlow12 Can we add an issue to resolve these test skips so we make sure we don't forget about them?